liip/imagine-bundle/1.8/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Liip\\ImagineBundle\\LiipImagineBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/"
+    },
+    "copy-from-package": {
+    	"Resources/config/routing.yaml": "%ETC_DIR%/routing/imagine.yaml"
```

Comment:
why not simply importing the route from the bundle itself ? This way, people won't alter the route definitions by mistake (making it a maintenance nightmare for you as you won't know what the actual content of the file is).

Btw @fabpot, is the file copied from the package updated when updating the package later ? I don't think so. So this would be another argument for importing properly.